GenderPayGap.WebUI/Views/ReportSizeOfOrganisation/NewReportSizeOfOrganisation.cshtml

```diff
@@ -0,0 +1,102 @@
+﻿
+@using GenderPayGap.Core.Helpers
+@using GenderPayGap.WebUI.Views.Components.StatusMessages
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@model GenderPayGap.WebUI.Models.Report.ReportSizeOfOrganisationViewModel
```

Comment:
we should change the labels to match the old ones:
-  "Less than 250, I wish to report voluntarily"
- "-" instead of "to"